src/Sylius/Bundle/CoreBundle/Command/InstallSampleDataCommand.php

```diff
@@ -43,16 +45,18 @@ protected function execute(InputInterface $input, OutputInterface $output): ?int
     {
         /** @var QuestionHelper $questionHelper */
         $questionHelper = $this->getHelper('question');
+        $suite = $input->getOption('fixture-suite');
 
         $outputStyle = new SymfonyStyle($input, $output);
         $outputStyle->newLine();
         $outputStyle->writeln(sprintf(
-            'Loading sample data for environment <info>%s</info>.',
-            $this->getEnvironment()
+            'Loading sample data for environment <info>%s</info> from suite <info>%s</info>.',
+            $this->getEnvironment(),
+            $suite
```

Comment:
We should probably have some default value here? `$suite ?? 'default'`?

Reply:
You're right